### PR TITLE
Fix 400+ typos

### PR DESCRIPTION
**Pull request by Lioness100**
Let me know if there's anything I need to fix :)

fixes #1795

### CODE REVIEW

**Comment by paxperscientiam** [comment]
"All committers have signed the CLA." lol

**Comment by pierizvi** [comment]
LGTM

**Comment by insomnia-creator** [comment]
LGTM, please merge.

**Comment by thearyannarula** [comment]
LGTM

**Comment by realKarthikNair** [comment]
LGTM

**Comment by Dev-R** [comment]
LGTM

**Comment by porterclev** [comment]
^

**Comment by saurvkmr** [comment]
what IDE they use at twitter? generally Intellij prompts typos!!

**Comment by WilliamDavidHarrison** [comment]
This is one of the only PR's that is actually legit. LGTM!

File: visibilitylib/src/main/scala/com/twitter/visibility/rules/Action.scala
Comment (fanfare) [inline visibilitylib/src/main/scala/com/twitter/visibility/rules/Action.scala:1]: line `659` still contains the typo `toTweetVisbilityNudgeThrift` 

    tweetVisibilityNudge.map(_.toTweetVisbilityNudgeThrift()),

Comment (Lioness100) [reply visibilitylib/src/main/scala/com/twitter/visibility/rules/Action.scala:1]: Oops, good catch, I'll fix that today

**Review by pierizvi — COMMENTED** [review]
LGTM

File: visibilitylib/src/main/scala/com/twitter/visibility/configapi/configs/DeciderKey.scala [hunk 1]
```diff
@@ -1004,7 +1004,7 @@ private[visibility] object DeciderKey extends DeciderKeyEnum {
     "visibility_library_enable_localized_interstitial_generator"
   )
 
-  val EnableProfileMixeMediaSafetyLevel: Value = Value(
+  val EnableProfileMixerMediaSafetyLevel: Value = Value(
```
Comment (fanfare) [inline visibilitylib/src/main/scala/com/twitter/visibility/configapi/configs/DeciderKey.scala:1007]: here is another one, check line `115` in [`scala/com/twitter/visibility/configapi/configs/VisibilityDeciders.scala`](https://github.com/twitter/the-algorithm/blob/138bb519975407d4ea0dc1478d897d451ef05dab/visibilitylib/src/main/scala/com/twitter/visibility/configapi/configs/VisibilityDeciders.scala) which still has the typo `EnableProfileMixeMediaSafetyLevel`

        ProfileMixerMedia -> DeciderKey.EnableProfileMixeMediaSafetyLevel,

**Comment by fernandoxu** [comment]
LGTM!!

File: visibilitylib/src/main/scala/com/twitter/visibility/configapi/configs/DeciderKey.scala (hunk 1 above)
Comment (Lioness100) [reply visibilitylib/src/main/scala/com/twitter/visibility/configapi/configs/DeciderKey.scala:1007]: Tysm!!

**Review by mammadyahyayev — COMMENTED** [review]
LGTM

**Review by fernandoxu — COMMENTED** [review]
LGTM

**Review by xhyrom — COMMENTED** [review]
🤔🙂 LGTM

**Comment by RaghavGoel01** [comment]
LGTM

**Comment by Soulter** [comment]
LGTM

**Review by ravimourya — COMMENTED** [review]
reviewd. looks good

**Review by harshsingh32 — COMMENTED** [review]
looks good

**Review by harshsingh32 — COMMENTED** [review]
looks good

**Review by acenoor — COMMENTED** [review]
`lgtm`

**Review by dromzeh — COMMENTED** [review]
LGTM

**Comment by Lioness100** [comment]
It's been real 🫡

**Pull request closed without merging**